frcnn_detector.py

```diff
@@ -18,13 +18,13 @@
 roi_maxDimRel = 0.4
 roi_minNrPixelsRel = 2 * roi_minDimRel * roi_minDimRel
 roi_maxNrPixelsRel = 0.33 * roi_maxDimRel * roi_maxDimRel
-roi_maxAspectRatio = 4.0  # maximum aspect Ratio of a ROI vertically and horizontally
+roi_maxAspectRatio = 7.0  # maximum aspect Ratio of a ROI vertically and horizontally

```

Comment:
Is this related to the change? If it fits a certain scenario I rather think of a better way that we can set this as parameter.